.github/workflows/reusable_jira_link.yaml

```diff
@@ -125,10 +125,10 @@ jobs:
               echo "JIRA_RESPONSE: $JIRA_RESPONSE"
```

Comment:
It won't let me add comments to lines outside of the range of changes, but lets remove the 'X' character from the regex's on lines 36 and 61 as well so DEV-XXXXXX doesn't pass the initial parse either

Reply:
Done. It's in my new PR. Closed this PR due to bad branch name anyway.